### ISSUE TITLE
Implement support to the RESOURCE variable in libmodsecurity

### ISSUE THREAD

**Issue by zimmerle** [SpiderLabs/ModSecurity#1014]
The variable RESOURCE needs to be implemented inside libModSecurity:

More information about this variable can be found at ModSecurity reference manual: https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual

**Comment by victorhora** [comment]
Hey @zimmerle I'm wondering if this one is meant to be for the [RESOURCE](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual#setrsc) collection or for a possibly undocumented RESOURCE variable?

Because if I'm not mistaken the collection seems to be already working as you can use the following set of rules to initialize a resource collection:
```
SecRule ARGS:resource "@unconditionalmatch " "phase:2,pass,initcol:resource=%{ARGS.resource},id:900003"
SecRule ARGS:resource "@unconditionalmatch " "phase:2,pass,expirevar:resource.timeout=3600,id:9000033"
```

So if this is true, we should change this milestone to something like [setrsc action is implemented](https://github.com/SpiderLabs/ModSecurity/blob/v3/master/src/parser/seclang-parser.yy#L2183-L2186)? :)

**Comment by zimmerle** [comment]
Fully implemented as of: 9369efcb90229a8e5a4eb410f7d7db0691c56f62
